### PR TITLE
Add molmass

### PR DESCRIPTION
**Pull request by RJMW**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/molmass/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - molmass = molmass.molmass:main
+    - molmass_web = molmass.molmass_web:main
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (bgruening) [inline recipes/molmass/meta.yaml]: pip install ...

File: recipes/molmass/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+requirements:
+  host:
+    - python
+    - setuptools
```
Comment (bgruening) [inline recipes/molmass/meta.yaml]: adding pip

File: recipes/molmass/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    - molmass --help
+    - molmass_web --help
+
+about:
+  home: https://www.lfd.uci.edu/~gohlke/
+  license: BSD License
+  license_family: BSD
+  license_file: ''
+  summary: Molecular Mass Calculations
+  description: "Molecular Mass Calculations\n===========================\n\nMolmass is a Python library and console script to calculate the molecular mass\n(average, nominal, and isotopic pure), the elemental\
```
Comment (bgruening) [inline recipes/molmass/meta.yaml]: this could be cleanuped up a little bit

File: recipes/molmass/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    - molmass
+  commands:
+    - molmass --help
+    - molmass_web --help
+
+about:
+  home: https://www.lfd.uci.edu/~gohlke/
+  license: BSD License
+  license_family: BSD
+  license_file: ''
```
Comment (bgruening) [inline recipes/molmass/meta.yaml]: is there a license file

File: recipes/molmass/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    \    Move modules into molmass package.\n2018.5.29\n    Add option to start web interface from console.\n2018.5.25\n    Style and docstring fixes.\n    Make 'from_fractions' output deterministic.\n\
+    2005.x.x\n    Initial release.\n\nExamples\n--------\n>>> from molmass import Formula\n>>> f = Formula('D2O')  # heavy water\n>>> f.formula  # hill notation\n'[2H]2O'\n>>> f.empirical\n'[2H]2O'\n>>>\
+    \ f.mass  # average mass\n20.027603556\n>>> f.isotope.massnumber  # nominal mass\n20\n>>> f.isotope.mass  # monoisotopic mass\n20.0231181781\n>>> f.atoms\n3\n>>> print(f.composition())\nElement  Number\
+    \  Relative mass  Fraction %\n2H            2       4.028204     20.1133\nO             1      15.999400     79.8867\nTotal:        3      20.027604    100.0000\n>>> print(f.spectrum())\nRelative mass\
+    \    Fraction %      Intensity\n20.023118         99.757000     100.000000\n21.027335          0.038000       0.038093\n22.027364          0.205000       0.205499\n\n\n"
+  doc_url: ''
```
Comment (bgruening) [inline recipes/molmass/meta.yaml]: remove?

File: recipes/molmass/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  dev_url: ''
+
+extra:
+  recipe-maintainers: ''
```
Comment (bgruening) [inline recipes/molmass/meta.yaml]: add yourself?

**Comment by daler** [comment]
Closing due to inactivity, please re-open if needed.

**Pull request closed without merging**